docs/content/docs/engines/build.md

```diff
@@ -50,6 +50,12 @@ You can find Flink 1.15 bundled jar in `./flink-table-store-dist/target/flink-ta
 
 To build with Hive 2.3, no special argument is needed.
 
+To build with Hive 3.1, run the following command.
```

Comment:
IMO, it's better to add a table to show the mapping between the Hive version and the corresponding command.

Reply:
Good point!